src/View/Helper/FormHelper.php

```diff
@@ -63,8 +63,8 @@ class FormHelper extends Helper
      */
     protected $_templateSet = [
         'default' => [
-            'checkboxContainer' => '<div class="form-check"{{required}}>{{content}}{{help}}</div>',
-            'checkboxContainerError' => '<div class="form-check">{{content}}{{error}}{{help}}</div>',
+            'checkboxContainer' => '<div class="form-group {{type}}{{required}}">{{content}}{{help}}</div>',
```

Comment:
This should stay `form-check`, but it's related to the nesting problem, so, guess that can be addressed later on.

Reply:
Yes, it's related to the nesting so haven't made any change related to that.